src/index.html

```diff
@@ -4,10 +4,48 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="stylesheet" href="./../src/styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+
+    <div class="product-card" data-qa="card">
+
+      <div class="photo"></div>
+
+      <div class="description">
+        <div class="description__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+        <div class="description__product-code">Product code: 195434</div>
+      </div>
+
+      <div class="rating">
+
+        <div class="rating__stars">
+          <div class="rating__stars rating__stars--4">
```

Comment:
```suggestion
          <div class="rating__stars stars stars--4">
```
stars should be the separate bem block